docs/installation/network-options.md

```diff
@@ -18,39 +18,57 @@ This page describes the following networking options:
 
 ## Flannel Options
 
-The default backend for flannel is VXLAN. To enable encryption, pass the IPSec (Internet Protocol Security) or WireGuard options below.
+[Flannel](https://github.com/flannel-io/flannel/blob/master/README.md) is a lightweight provider of layer 3 network fabric that implements the Kubernetes Container Network Interface (CNI). It is what is commonly referred to as a CNI Plugin.
 
-If you wish to use WireGuard as your flannel backend it may require additional kernel modules. Please see the [WireGuard Install Guide](https://www.wireguard.com/install/) for details. The WireGuard install steps will ensure the appropriate kernel modules are installed for your operating system. You need to install WireGuard on every node, both server and agents before attempting to leverage the WireGuard flannel backend option.
-The `wireguard` backend will be removed from v1.26 in favor of `wireguard-native` backend natively from Flannel.
+* Flannel options can only be set on server nodes, and must be identical on all servers in the cluster.
+* The default backend for Flannel is `vxlan`. To enable encryption, use the `wireguard-native` backend.
+* Using `vxlan` on Rasperry Pi with recent versions of Ubuntu requires installation of the `linux-modules-extra-raspi` package. ([#4234](https://github.com/k3s-io/k3s/issues/4234#issuecomment-951993944))
```

Comment:
```suggestion
* Using `vxlan` on Rasperry Pi with Ubuntu 21.04+ requires [additional preparation.](https://docs.k3s.io/advanced#enabling-vxlan-for-ubuntu-2110-on-raspberry-pi)
```
We already have a section in the docs devoted to this, I would prefer linking to that rather than the GH issue.